pkg/oci/build.go

```diff
@@ -269,7 +275,7 @@ func createManifestForTarget(ctx context.Context, target oras.Target, metadataFi
 	manifestDesc := content.NewDescriptorFromBytes(ocispec.MediaTypeImageManifest, manifestJson)
 	manifestDesc.Platform = &ocispec.Platform{
 		Architecture: arch,
-		OS:           "linux",
+		OS:           "ebpf",
```

Comment:
This is to avoid docker trying to run gadgets. 

Before this change 
```bash 
$ docker run --pull always ghcr.io/inspektor-gadget/gadget/trace_open:v0.32.0
v0.32.0: Pulling from inspektor-gadget/gadget/trace_open
docker: unsupported media type application/vnd.gadget.config.v1+yaml.
See 'docker run --help'.
```

after this change


```bash 
$ docker run --pull always 127.0.0.1:5000/hello:latest
latest: Pulling from hello
docker: no matching manifest for linux/amd64 in the manifest list entries.
See 'docker run --help'.
```